modules/nf-core/sratools/prefetch/templates/retry_with_backoff.sh

```diff
@@ -47,7 +47,16 @@ retry_with_backoff !{args2} \
     !{args} \
     !{id}
 
-[ -f !{id}.sralite ] && vdb-validate !{id}.sralite || vdb-validate !{id}
+# check file integrity using md5sum or vdb-validate as fallback
+MD5SUM=$(
+	curl -s 'https://locate.ncbi.nlm.nih.gov/sdl/2/retrieve?filetype=run&acc=!{id}' |
```

Comment:
You are making another network request, please make the curl command more robust by adding options for failing `--fail` and retrying.